### PR TITLE
UG documentation for the toolbar in ImageEditor

### PR DESCRIPTION
**Pull request by SriRadheshNagS**
UG documentation for the toolbar in ImageEditor

Added the tool bar text customization

### CODE REVIEW

File: MAUI/ImageEditor/events.md [hunk 1]
```diff
@@ -64,6 +64,56 @@ N> This is common for Shape and Text annotations.
 
 {% endtabs %}
 
+## ID Support for annotation event
```
Comment (saiganesh-sakthivel) [inline MAUI/ImageEditor/events.md:67]: ## ID support for annotation
In ImageEditor, a unique ID is generated for all annotations (Text, Shapes, Paths, and CustomViews) when they are added to the image editor. You can retrieve this unique ID from the ItemsSelected event arguments or from the serialized JSON.

## Select a particular annotation programmatically using annotation ID
By passing the unique ID of an annotation to the SelectAnnotation method of SfImageEditor, you can select the particular annotation programmatically.

**Review by saiganesh-sakthivel — APPROVED** [review]
fine

File: MAUI/ImageEditor/events.md [hunk 2]
```diff
@@ -64,6 +64,60 @@ N> This is common for Shape and Text annotations.
 
 {% endtabs %}
 
+## ID Support for annotation
```
Comment (Jeyasri-Murugan) [inline MAUI/ImageEditor/events.md:67]: Move under Text and Shape title

**Review by saiganesh-sakthivel — APPROVED** [review]
fine

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**